crates/e2e/Cargo.toml

```diff
@@ -10,6 +10,7 @@ app-data = { path = "../app-data" }
 anyhow = { workspace = true }
 async-trait = { workspace = true }
 autopilot = { path = "../autopilot" }
+axum = { workspace = true }
```

Comment:
I'm just curious, why not warp, which is already used in the `ZeroExApi` mock?

Reply:
I'd like to avoid using `wrap` at all costs. The `wrap` crate made sense 2 years ago when `axum` was in a very early stage and didn't implement many features. Nowadays, `axum` has everything which `wrap` has, but better.

Reply:
Yeah, but since `axum` is used only to implement dummy mocks, 99% of its futures won't be used and its configuration seems more involved.

Reply:
Majority of the configuration in `axum` is not really needed. I just deleted it for simplicity.